src/placeos-models/user.cr

```diff
@@ -324,3 +331,20 @@ module PlaceOS::Model
     end
   end
 end
+
+# monkey patch support for the required query
+module RethinkDB
```

Comment:
before we merge do we want to move this monkey patch into the base Rethink lib?

Reply:
Definitely.

Reply:
/ take it out of here. Not in use anymore.